scanner/Makefile

```diff
@@ -50,8 +54,8 @@ all: deps scanner-go-build images
 ###############################################################
 
 OSSLS_BIN := $(GOBIN)/ossls
-$(OSSLS_BIN):
-	$(SILENT)echo "+ $@"
```

Comment:
I understand its use. But the command in question `echo "+ $@"` would be printed twice in the output if `SILENT=`, and that's redundant.

`echo`s should generally be prefixed with `@` in rules since printing the echo command and followed by its output is redundant.

Reply:
Also, I think I mentioned this before: $(SILENT) seems an anti-pattern considering `make` offers `.SILENT` and some other related toggles.